server/internals/service/service.go

```diff
@@ -19,11 +19,21 @@ import (
 	rs_utils "globe-and-citizen/layer8/server/resource_server/utils"
 )
 
+type ServiceInterface interface {
```

Comment:
I like it. In general, coding to an interface is a good way to avoid many future headaches.